Abstract Syntax and Semantics Document/index.html

```diff
@@ -156,40 +156,57 @@
     <h4>Instants and Intervals</h4>
     
     <p>Following the concepts of the Time Ontology (<a href="http://www.w3.org/TR/owl-time/">http://www.w3.org/TR/owl-time/</a>), a temporal entity can be a time instant or a time interval.</p>
-    
+    <section id="def-timestamped-graph">
     <h3>Timestamped graph</h3>
     
-    <p>A <em>timestamped graph</em> is defined as an RDF Dataset under the RDF Dataset semantics that <a href="http://www.w3.org/TR/2014/NOTE-rdf11-datasets-20140225/#each-named-graph-defines-its-own-context">each graph defines its own context</a> with the following constraints.</p>
+      <p>A <em>timestamped graph</em> is defined as an RDF Dataset under the RDF Dataset semantics that <a href="http://www.w3.org/TR/2014/NOTE-rdf11-datasets-20140225/#each-named-graph-defines-its-own-context">each graph defines its own context</a> and where a particular triple in the default graph has been designated as the timestamp triple, with the following constraints.</p>
```

Comment:
<p>A <em>timestamped graph</em> is defined as an RDF Dataset under the RDF Dataset semantics that 

== maybe better to change to ==> 

<p>A <em>timestamped graph</em> is defined to be an <a href="http://www.w3.org/TR/2014/NOTE-rdf11-datasets-20140225/#each-named-graph-defines-its-own-context"> RDF dataset </a> with a temporal context under the following constraints: